src/components/opoTable/opoTable.js

```diff
@@ -14,88 +16,137 @@ export default function OpoTable({
   opos,
   title,
 }) {
-  if (!opos.length) return null;
+  const columns = useMemo(() => {
+    const cols = inState
+      ? ["name", "region", "tier", "donors", "shadow", "investigation"]
+      : ["states", "name", "tier", "donors", "shadow"];
 
-  const Heading = ({ color, title }) => (
-    <th scope="col" className={color ?? null}>
-      {headings[title].heading}
-      {citations[title] && (
-        <sup>
-          <a
-            className={color ?? null}
-            href={`#citations-${citations[title].index}`}
-            target="_self"
-          >
-            {citations[title].index + 1}
-          </a>
-        </sup>
-      )}
-    </th>
-  );
+    const Heading = ({ color, title }) => (
+      <div>
+        {headings[title].heading}
+        {citations[title] && (
+          <sup>
+            <a
+              className={color ?? null}
+              href={`#citations-${citations[title].index}`}
+              target="_self"
+            >
+              {citations[title].index + 1}
+            </a>
+          </sup>
+        )}
+      </div>
+    );
+    const createCol = accessor => {
+      let col = {
+        Header: (
+          <Heading
+            title={accessor}
+            color={accessor === "shadow" ? "red" : null}
+          />
+        ),
+        accessor,
+      };
+      if (accessor === "donors" || accessor === "investigation")
+        col.cellClass = "text-center";
+      if (accessor === "shadow") {
+        col.cellClass = styles.shadows;
+        col.color = "red";
+      }
+      if (accessor === "tier") {
+        col.Cell = props => (
+          <Container>
+            <Tier className={styles.tierCol} tier={props.value} />
+          </Container>
+        );
+        col.cellClass = styles.tierCol;
+      }
+      return col;
```

Comment:
nit: let's add `else`s and `returns` here, since they're mutually-exclusive anyway. In combo with the `const` change, for exmaple:
```
if (accessor === "donors" || accessor === "investigation") {
  return {
    ...col,
    cellClass: "text-center"
  };
} else if (accessor === "shadow") {
// etc
```

and `else { return col; }` later